apps/examples/testcase/le_tc/drivers/tc_bch.c

```diff
@@ -0,0 +1,341 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_bch.c
+/// @brief Test Case Example for bch driver
+#include <tinyara/config.h>
+#include <tinyara/fs/ioctl.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include "tc_internal.h"
+#include <errno.h>
+
+#define SECT_SIZE	512
+
+/* Closes all open file descriptors */
+static void close_fds(int *fds, int count)
```

Comment:
Same as below.

Reply:
I cannot use #define here. Compilation error due to use of pre-increment operator during call. Instead I made the function inline.